src/main/cpp/tiv.cpp

```diff
@@ -382,6 +382,26 @@ void emit_image(const cimg_library::CImg<unsigned char> & image, int flags) {
 }
 
 
+struct size {
+  size(unsigned int in_width, unsigned int in_height) :
+	width(in_width), height(in_height) {
+  }
+  size(cimg_library::CImg<unsigned int> img) :
+	width(img.width()), height(img.height()) {
+  }
+  unsigned int width;
+  unsigned int height;
+};
+size operator*(size lhs, double scale) {
```

Comment:
Shouldn't lhs be const&? 

Why not make this a method "scale" of size (keeping the original, returning a new instance)

Reply:
No particular reason to make it global instead of a method. I've made and pushed the change. I suppose that also eliminates the question about passing by const &. :-)

Reply:
Oh, I misread; you were proposing calling it "scale". This seems like an reasonable use of operator overloading to me, To me the intent of `size(2,3)*4` seems obvious. I'm happy to change if you disagree.

Reply:
Sorry, some background: The company I work for has a certain code style where operator overloading is forbidden. So for me it's easier to maintain code if it follows this style consistently. That said, I value contributions very much and if you strongly prefer the operator I am fine with that.